packages/exe/setup.js

```diff
@@ -1,25 +1,25 @@
 const path = require('path')
 const fs = require('fs')
 
-const platform = process.platform == 'win32'
+const platform = process.platform === 'win32'
   ? 'win'
-  : process.platform == 'darwin'
+  : process.platform === 'darwin'
   ? 'macos'
   : process.platform
-const arch = platform == 'win' && process.arch == 'ia32' ? 'x86' : process.arch
+const arch = platform === 'win' && process.arch === 'ia32' ? 'x86' : process.arch
 
 const pkgName = `@pnpm/${platform}-${arch}`
 const pkgJson = require.resolve(`${pkgName}/package.json`)
 const subpkg = JSON.parse(fs.readFileSync(pkgJson, 'utf8'))
 
-if (subpkg.bin != null) {
+if (subpkg.bin !== null) {
```

Comment:
no, this is not good. `!=` should be used because it will work with both undefined and null

Reply:
Yes, when I saw this, I was thinking: "There is only `null` in JSON, there will be no `undefined`", but I didn't realize until just now that if there is no corresponding property, the parsed value is also `undefined`.

Reply:
So, here, can we just check if the value is truthy? like this:

```javascript
if (subpkg.bin) {
  // ...
}
```

Reply:
I don't think the linter will allow this. And I also don't see value in this change.

Reply:
Yes, I withdraw this modification.